src/ast/table.rs

```diff
@@ -126,6 +127,86 @@ impl<'a> Table<'a> {
 
         Ok(result)
     }
+
+    pub fn left_join<J>(mut self, join: J) -> Self
```

Comment:
This needs a public documentation with examples. Read `Select` for info how they should be written.